peekingduck/pipeline/nodes/output/utils/csvlogger.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utils for CSV logging
+"""
+import csv
+from datetime import datetime
+from typing import Any, Dict, List
+
+class CSVLogger:
+
+    def __init__(self,filepath: str,headers: List[str],logging_interval: int=1) -> None:
```

Comment:
Minor styling - would be good to have a space after commas.

Reply:
Thanks for the reminder, will take note in future

Reply:
Thanks for the change